examples/getting-started-typescript/tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "resolveJsonModule": true,
     "allowSyntheticDefaultImports": true,
     "allowJs": true,
+    "skipLibCheck": false,
```

Comment:
This overrides the default `extend` value: https://www.npmjs.com/package/@tsconfig/node18